### PR TITLE
Feature/cli maker lp stats

### PR DESCRIPTION
**Pull request by OlaStenberg**
`scli maker`
image

`scli maker -n avalanche`
image

Missing networks:
`MOONBEAM`: reserveUSD returns 0
`HECO`: Subgraph not working, not much traffic
`HARMONY`: inconsistent uptime, not hosted by thegraph
`OKEX`: Subgraph not working, not much traffic

We should investigate moonbeam and see why the pricing isn't working - and we need a stable harmony subgraph, otherwise we can't consistently build the cli since `generate` command tries to build the schema from the api.

### CODE REVIEW

**Comment by LufyCZ** [comment]
For Harmony - The TheGraph team has been promising support for it for a quite a while now, we're blocked by them fully.

File: packages/cli/graph/graph-client.ts [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+    },
+    // {
+    //   network: [ChainKey.MOONBEAM],
+    //   address: CHAIN_CONFIG[ChainKey.MOONBEAM].address,
+    //   type: CHAIN_CONFIG[ChainKey.MOONBEAM].type,
+    //   liquidityPositions: (await sdk.MoonbeamUserLPs({ id: CHAIN_CONFIG[ChainKey.MOONBEAM].address })).MOONBEAM_EXCHANGE_user
+    //     ?.liquidityPositions,
+    // },
+  ]
+}
```
Comment (LufyCZ) [inline packages/cli/graph/graph-client.ts:160]: Convert to use `Promise.all` for perf benefits

**Comment by OlaStenberg** [comment]
Think I was on node 14 which makes the builds fail, will look at it later today

File: packages/cli/index.ts [hunk 2]
```diff
@@ -13,6 +13,10 @@ program
   .option('-a,--all', 'all', false)
   .action(chef)
 
+program.command('maker').description('get LPs in USD across all networks, or specify network to get more detail information')
+.option('-n,--network <NETWORK>', 'network')
+.action(maker)
```
[the remark this answers is not in the document]
Comment (OlaStenberg) [reply packages/cli/index.ts]: Good idea, added verbose flag

It spits out so many lines, I recommend piping the response to a file when running this

File: packages/cli/graph/graph-client.ts [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+    ).CELO_EXCHANGE_user?.liquidityPositions
+  } else if (chain === ChainKey.MOONRIVER) {
+    return await (
+      await sdk.MoonriverUserLPs({ id: CHAIN_CONFIG[chain].address })
+    ).MOONRIVER_EXCHANGE_user?.liquidityPositions
+  } else if (chain === ChainKey.FUSE) {
+    return await (
+      await sdk.FuseUserLPs({ id: CHAIN_CONFIG[chain].address })
+    ).FUSE_EXCHANGE_user?.liquidityPositions
+  }
```
[the remark this answers is not in the document]
Comment (OlaStenberg) [reply packages/cli/graph/graph-client.ts]: Yeah I know, this isn't pretty at all. I had a go at what you suggested, but the responses were undefined when trying casting them to `EthereumUserLPs`. I renamed it to `MAKER_CONFIG` and moved it to `config.ts`, it looked weird to have a graph client calls in there. If you wanna have a go at this, feel free to make commits to this branch

**Pull request merged**